### ISSUE TITLE
Pipeline feature request

### ISSUE THREAD

**Issue by dio99** [Graylog2/graylog2-server#6088]
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Graylog Version:
* Elasticsearch Version:
* MongoDB Version:
* Operating System:
* Browser version:

**Comment by dio99** [comment]
Hi i would like to have the pipeline in vendor group, so u dont get a very long list of rules
and also be able to when u add a rule to a pipeline that u can add that group/vendor and not have to click on all 100 rules to be added..

graylog version 3.02

// Anders

**Comment by jalogisch** [comment]
he @dio99 

do I understand you right that you want to be able to group the processing pipeline rules?

**Comment by dio99** [comment]
yes i would like to have in pipelines rules groups and also be able to add the group to a pipeline
as u know now i have to choose in dropdown list and if u have 100 rules u are not sure if u added all or double added some rules.
